### ISSUE TITLE
New open source community forum! www.sharetribe.com/community/

### ISSUE THREAD

**Issue by rap1ds** [sharetribe/sharetribe#1806]
Hello everyone!

We have set up a new discussion forum for Sharetribe open source community! It's a [Discourse](https://www.discourse.org/) forum at [www.sharetribe.com/community/](https://www.sharetribe.com/community/).

We hope to see the forum growing actively! Please go and register and start discussing!

So when should you add a new Issue to Github, and when post a new thread to Discourse?

The rule of thumb is you should add a new Issue **only for reproducible bugs in Sharetribe code and documentation.** All support questions should go to forum.

I hope to see you on the forum!
## 

Mikko from Sharetribe

**Comment by rap1ds** [comment]
Some background:

Before the forum, we had a Flowdock channel for open-source community. The chat room never felt like the best solution, and we have now decided to switch to a community forum.

Switching to a forum gives us many benefits:

* It is easier to find existing answers. Due to the nature of chat, we noticed that the same questions kept coming up over and over again. With a forum, finding old questions and answers is a lot easier.
* Googlability. The forum discussions are public, meaning people can find Sharetribe help using a search engine.
* Active users can build a reputation. Our forum software of choice, Discourse, allows those who answer lots of questions to advance in trust levels. This will help users identify community members with the most knowledge.

**Comment by nwnzz** [comment]
Hello,
Please help me, I installed all the things as per instruction in my rails folder on digitalocean server but unable to get sharetribe only apache ubuntu page is showing , you can check here http://198.199.123.48/
Thanks in advance please help me out.

**Comment by rap1ds** [comment]
@nwnzz Hi! As mentioned in this thread, the open source community forum (www.sharetribe.com/community/) is exactly the right place for asking help with installation! We want to keep Github issues for bug reports only.

Unfortunately, you need to give some more information about your problem. To me it seems that you're not even hitting the Rails server but instead some default Apache Ubuntu page on port 80. Have you tried port 3000?

Let's continue the discussion at the open source forum!

**Comment by nwnzz** [comment]
Thank you for your reply .
yes I tried http://198.199.123.48:3000/
but its showing webpage is not available

**Comment by zenikk** [comment]
@rap1ds

I see that for some reason you have switched https://www.sharetribe.com/community/ to be  a simplified static archive version.
Does it mean you are going to get rid only of Discourse or community forum at all?

**Comment by thomasmalbaux** [comment]
@zenikk Thanks for the message! Actually there is actually some issue with Discourse and we are in touch with their support to get things back to normal asap.

Stay tuned!

**Comment by aayushdutt** [comment]
Can't create an account in the community. It says 'New registrations are not allowed from your IP address'.
![image](https://user-images.githubusercontent.com/25280541/78020732-9c5b1380-736f-11ea-8ed5-02d6b2f15eb9.png)

**Comment by thomasmalbaux** [comment]
Thanks for reporting!

I've noticed that you've created an account at https://www.sharetribe.com/community/u/aayushdutt/summary. All good now?

Indeed, there has been lots of spam lately in that forum, some IP banning rules might be a bit too strong, thanks for sharing your experience!

**Comment by aayushdutt** [comment]
Thanks, it worked after a while. All working fine now.

**Comment by rpmcoding** [comment]
@thomasmalbaux I am getting the same issue with not being able to create an account on the forum.  It says, "New registrations are not allowed from your IP address".

If you're getting a lot of spam, perhaps add captcha, rather than blocking IP's too strictly.

**Comment by thomasmalbaux** [comment]
Thanks for reporting @rpmcoding! Unfortunately, the service provider we use to host that forum doesn't give us any control over this (they have their own setup for this) nor support reCaptcha.

Still, we can message them to report IPs that are wrongly blocked. I've noticed you contacted our team via email, let's continue the discussion there to get more details.

**Comment by shoesack** [comment]
Hey @thomasmalbaux ,

same happens to me for weeks now. Can't even login to my existing account (SHOEsack) nor create a new one. Always getting "You can't login as SHOEsack from that IP address." or "New registrations are not allowed from your IP address".

How to proceed with unblocking my IP / IP block?

**Comment by thomasmalbaux** [comment]
@shoesack Thanks for reporting! Would you share the IP address from which you tried logging into the Community forum? You can message us at hello@sharetribe.com.

I'll check if I can see anything wrong with it and will message the service provider we rely on for the forum hosting, see if they can unblock it.
